api/tests/src/api/users/test_user_route_token.py

```diff
@@ -25,3 +29,34 @@ def test_post_user_route_token_400(client, api_auth_token):
     resp = client.post("v1/users/token", headers={"X-Auth": api_auth_token})
     assert resp.status_code == 400
     assert resp.get_json()["message"] == "Missing X-OAuth-login-gov header"
+
+
+def test_post_user_route_token_refresh_200(
+    enable_factory_create, client, db_session, api_auth_token
+):
+    user = UserFactory.create()
+    token, user_token_session = create_jwt_for_user(user, db_session)
+    expiration = user_token_session.expires_at
+    db_session.commit()
+
+    resp = client.post("v1/users/token/refresh", headers={"X-SGG-Token": token})
+
+    db_session.refresh(user_token_session)
+
+    assert resp.status_code == 200
+    assert user_token_session.expires_at != expiration
```

Comment:
A trick for testing time related things is to use the freeze_time library (see an example here: https://github.com/HHS/simpler-grants-gov/blob/main/api/tests/src/auth/test_api_jwt_auth.py#L160 )

You can wrap a bit of code like a call to the API (or the [whole test](https://github.com/HHS/simpler-grants-gov/blob/main/api/tests/src/auth/test_api_jwt_auth.py#L57)) in a specific time, and then know that the expires_at will always be that time + 30.